packages/adblocker/src/request.ts

```diff
@@ -70,11 +70,44 @@ export type PlaywrightRequestType =
   | 'websocket'
   | 'xhr';
 
-// From: https://developer.chrome.com/extensions/webRequest#type-ResourceType
-export type WebRequestTypeChrome = chrome.webRequest.ResourceType;
+// From: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chrome/index.d.ts#L11989
```

Comment:
Let's use a permalink.
```suggestion
// From: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3867576ef95d769409f777091b29b4ea05f0b89a/types/chrome/index.d.ts#L11989
```